phac-nml/irida-next
# Fix: Sample selection refresh

## What does this PR do and why?
Fixes the displayed selected number of samples is more than the total number of samples, during a sample cloning or transferring.
Fixes #796.

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/8059827a-56e3-424b-a3da-97a9c3016b46)

## How to set up and validate locally
1. Follow the steps from #508 to do a large sample transfer.
2. Open another tab within the browser and navigate to the **project** samples page the samples are transferring to. 
3. Open another tab within the browser and navigate to the **group** samples page the samples are transferring to.
4. Hit the `Select all` button.
5. Verify the total samples is larger than the selected samples within the table footer.
6. Also, verify the pagination components updated.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
